lib/simplecov.rb

```diff
@@ -49,11 +49,44 @@ def start(profile = nil, &block)
     end
 
     #
+    # Finds files that were to be tracked but were not covered and initializes
+    # their coverage to zero, with an estimation of the line count.
+    #
+    def add_uncovered_files(result)
+      if @track_files_glob
```

Comment:
I thought about loading the files, but I was a bit afraid of potential side effects (such as syntax errors, or weird metaprogramming stuff). Though that would be a point in favor of considering each line as not covered, as the file hasn't even been syntax checked. I'm OK with removing those checks and mark every line as not covered. Or would you prefer to load the files?

Personally I would rather have them in their original groups, otherwise I would probably tend to ignore them.